tools/python/util/__init__.py

```diff
@@ -4,12 +4,16 @@
 from .get_azcopy import get_azcopy
 from .logger import get_logger
 from .platform_helpers import (is_windows, is_macOS, is_linux)
-# Test what is needed here to use in a script
-# from .pytorch_export_helpers import infer_input_info
 from .run import run
 
 try:
     import flatbuffers  # noqa
     from .reduced_build_config_parser import parse_config
 except ImportError:
     get_logger('tools_python_utils').info('flatbuffers module is not installed. parse_config will not be available')
+
+# see if we can make the pytorch helpers available.
+import importlib.util  # noqa
```

Comment:
one idea, don't need it in this PR
could we include `__import__append.py` somehow instead of replicating the content?
something like `from .__import__append import *`?
perhaps `__import__append` could be named `ort_package_imports` or something to that effect

Reply:
Sounds good. We should definitely refine this setup on master.